src/app/containers/App/App.jsx

```diff
@@ -31,6 +31,17 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
   });
 
   const isInitialMount = useRef(true);
+  const shouldSetFocus = useRef(false);
+
+  useEffect(() => {
+    if (shouldSetFocus.current) {
```

Comment:
I think we could isolate some things inside the if block and test it in isolation, we could mock a `document` with a `h1` and test that running this actually sets the focus.
ie.
```
const setPageFocus = () => {
       const contentEl = document.querySelector('h1#content');
      if (contentEl) {
        contentEl.focus();
      }
};
```